pkg/ctl/cmdutils/pod_identity_association.go

```diff
@@ -173,12 +175,18 @@ type UpdatePodIdentityAssociationOptions struct {
 	PodIdentityAssociationOptions
 	// RoleARN is the IAM role ARN to be associated with the pod.
 	RoleARN string
+	// TargetRoleARN is the new target IAM role to associate with the service account.
+	// This is used for cross-account pod identity access.
+	TargetRoleARN string
+	// DisableSessionTags is a boolean flag to enable or disable session tags.
+	// This is used for cross-account pod identity access.
+	DisableSessionTags bool
```

Comment:
Maybe only this needs to be updated to a pointer and not the main API type since the main api type isn't used for updates?